src/storage/test/QueryEdgePropsTest.cpp

```diff
@@ -60,58 +60,36 @@ void buildRequest(cpp2::EdgePropRequest& req) {
         }
     }
     req.set_parts(std::move(tmpEdges));
-    req.set_edge_type(101);
     // Return edge props col_0, col_2, col_4 ... col_18
     decltype(req.return_columns) tmpColumns;
     for (int i = 0; i < 10; i++) {
-        tmpColumns.emplace_back(TestUtils::propDef(cpp2::PropOwner::EDGE,
-                                                   folly::stringPrintf("col_%d", i*2)));
+        tmpColumns.emplace_back(TestUtils::edgePropDef(folly::stringPrintf("col_%d", i * 2), 101));
     }
     req.set_return_columns(std::move(tmpColumns));
 }
 
 
 void checkResponse(cpp2::EdgePropResponse& resp) {
     EXPECT_EQ(0, resp.result.failed_codes.size());
-    EXPECT_EQ(13, resp.schema.columns.size());
+    EXPECT_EQ(10, resp.schema.columns.size());
     auto provider = std::make_shared<ResultSchemaProvider>(resp.schema);
     LOG(INFO) << "Check edge props...";
     RowSetReader rsReader(provider, resp.data);
     auto it = rsReader.begin();
     int32_t rowNum = 0;
     while (static_cast<bool>(it)) {
-        EXPECT_EQ(13, it->numFields());
-        {
-            // _src
-            // We can't ensure the order, so just check the srcId range.
-            int64_t v;
-            EXPECT_EQ(ResultType::SUCCEEDED, it->getInt<int64_t>(0, v));
-            CHECK_GE(30, v);
-            CHECK_LE(0, v);
-        }
-        {
-            // _rank
-            int64_t v;
-            EXPECT_EQ(ResultType::SUCCEEDED, it->getInt<int64_t>(1, v));
-            CHECK_EQ(rowNum % 7, v);
-        }
-        {
-            // _dst
-            int64_t v;
-            EXPECT_EQ(ResultType::SUCCEEDED, it->getInt<int64_t>(2, v));
-            CHECK_EQ(10001 + rowNum % 7, v);
-        }
+        EXPECT_EQ(10, it->numFields());
```

Comment:
Why change the UTs?

Reply:
Because for the query engine, there is no need to return the default properties of the  edges.

Reply:
I don't know why you have this conclusion.  The default props have been talked about.  Don't change this logic if it has no relationship with this pr.  If you have some thoughts about this place,  please open another pr for it.

Reply:
Here are two reasons:
1 The GO statement (example: go from 1 over 2) was to set _dst as the return prop to request the storage to return.
2 For the query edge prop, it does not use the default property returned. @CPWstatic

Reply:
To be faster,  rollback the changes about QueryEdgeProps,   it has no relationship with this pr and i don't think it is good enough to be merged in.

Reply:
Ok, I will rollback this.